posts/2023-09-19-23.0.0.9.adoc

```diff
@@ -74,18 +74,18 @@ Or take a look at our link:{url-prefix}/start/[Downloads page].
 [link=https://stackoverflow.com/tags/open-liberty]
 image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
 
-// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
 // Blog issue: https://github.com/OpenLiberty/open-liberty/issues/25973
 // Contact/Reviewer: hlhoots
-// // // // // // // // 
+// // // // // // // //
 [#sb3]
 == Spring Boot 3.0 support
 
-The Liberty Spring Boot Support 3.0 feature provides more complete support for running a Spring Boot 3.0 application on Liberty. It also provides the capability to "thin" the application when you create applications in containers. 
+The Liberty Spring Boot Support 3.0 feature supports running a Spring Boot 3.0 application on Liberty. It also provides the capability to link:{url-prefix}/docs/latest/deploy-spring-boot.html#thin[thin the application] when you create applications in containers.
```

Comment:
I think I prefer the (albeit more verbose) pervious wording.  Prior to this release, the runtime did provide some Spring Boot 3 feature support, but with this delivery, the support is being expanded.
```suggestion
The Liberty Spring Boot Support 3.0 feature provides a more complete support for running Spring Boot 3.0 applications on Liberty. It also provides the capability to link:{url-prefix}/docs/latest/deploy-spring-boot.html#thin[thin the application] when you create applications in containers.
```

Reply:
Yes, but as stated, that's not really clear and just adds more words. That said, I'm ok with leaving it as it was